crates/astria-composer/src/api.rs

```diff
@@ -20,29 +20,29 @@ use serde::Serialize;
 use tokio::sync::watch;
 use tracing::debug;
 
-use crate::searcher;
+use crate::composer_status::ComposerStatus;
 
 pub(super) type ApiServer = axum::Server<AddrIncoming, IntoMakeService<Router>>;
 
-type SearcherStatus = watch::Receiver<searcher::Status>;
+type ComposerStatusWatch = watch::Receiver<ComposerStatus>;
```

Comment:
Please no Hungarian notation (or anything that goes into that direction).
```suggestion
type ComposerStatus = watch::Receiver<ComposerStatus>;
```